example/neubot/client_dash.cpp

```diff
@@ -0,0 +1,54 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "src/common/utils.hpp"
+#include <functional>
+#include <iostream>
+#include <measurement_kit/common.hpp>
+#include <measurement_kit/ext.hpp>
+#include <measurement_kit/http.hpp>
+#include <measurement_kit/neubot.hpp>
+#include <stdlib.h>
+#include <string>
+#include <unistd.h>
+
+using namespace mk;
+using namespace mk::neubot::negotiate;
+using namespace mk::net;
+using namespace mk::http;
+
+static const char *kv_usage =
+    "usage: ./example/http/client_dash [-vn] [-a address]\n";
```

Comment:
Should be `./example/neubot/dash` (which in 0.3.0 would perhaps become the `neubot` command and the `dash` subcommand). Could you rename the file accordingly?